packages/frontapp/src/app/components/SearchWrapper/index.tsx

```diff
@@ -46,15 +47,18 @@ const SearchWrapper: FC<IProps> = ({ showSidebar }) => {
       {isReadersPage ? (
         <SearchByUserName />
       ) : (
-        <SearchByBookOrAuthor
-          path={
-            userRole === RolesTypes.READER
-              ? `${RoutesTypes.SEARCH}_by_name_or_author`
-              : RoutesTypes.BOOKS_STUFF
-          }
-        />
+        <>
+          <SearchByBookOrAuthor
+            path={
+              userRole === RolesTypes.READER
+                ? `${RoutesTypes.SEARCH}_by_name_or_author`
+                : RoutesTypes.BOOKS_STUFF
+            }
+          />
+        </>
       )}
       <FiltersButton />
+      <StatisticsButton />
```

Comment:
We should not show the statistics button for readers, render it only for managers